daemon-tests/src/lib.rs

```diff
@@ -276,8 +289,8 @@ impl Default for OpenCfdArgs {
     fn default() -> Self {
         Self {
             contract_symbol: ContractSymbol::BtcUsd,
+            initial_price: initial_price_for(ContractSymbol::BtcUsd),
```

Comment:
🙃 Could extract the `ContractSymbol` into a variable.

Reply:
indeed!